python/dgl/nn/pytorch/conv/gcn2conv.py

```diff
@@ -230,7 +232,11 @@ def forward(self, graph, feat, feat_0):
 
```

Comment:
Should edge weight also participate in symmetric adjacency normalization?

Reply:
https://github.com/pyg-team/pytorch_geometric/blob/74245f3a680c1f6fd1944623e47d9e677b43e827/torch_geometric/nn/conv/gcn_conv.py#L30-L65

I checked pyg's implementation. It uses the weight in the normalization part. Do you think it's reasonable?

Reply:
Personally I find this reasonable. Previously Tong has added `edge_weight` support to `GraphConv`, which does not perform normalization for weighted adjacency matrices. @hetong007 Do you find this reasonable? If so, what do you think of the backward compatibility for `edge_weight` in `GraphConv`?

Reply:
My PR didn't normalize the input because it accepts multi-dimensional weight vectors and there can be multiple ways to normalize it. Instead, I created a separate module that normalize a positive scalar edge weight as a standard implementation, and also let users to use customized normalization before feeding the edge-weight to the module.

Reply:
@hetong007 I see. Thanks for the clarification.
@VoVAllen I think [`EdgeWeightNorm`](https://docs.dgl.ai/api/python/nn.pytorch.html#dgl.nn.pytorch.conv.EdgeWeightNorm) already covers the cases we want. Please have a double check. If this is the case, then I guess we don't need to do anything :) We will however need to add a pointer to `EdgeWeightNorm` in the doc of these NN modules.